ci/courtesy-push.yml

```diff
@@ -0,0 +1,55 @@
+steps:
+- task: NuGetToolInstaller@0
+  inputs:
+    versionSpec: '5.4.0'
+  displayName: 'Install nuget'
+
+- task: DownloadBuildArtifacts@0
+  inputs:
+    artifactName: IndividualNugetPackages
+    downloadPath: 'IndividualNugetPackagesDownloaded'
+  displayName: 'Download Artifact'
+
+- script: dir
```

Comment:
I assume this was for testing, can we remove it? If it's helpful to keep for triage that is OK too.

Reply:
Whoops, yeah will remove